app/controllers/management/sites_controller.rb

```diff
@@ -38,8 +38,21 @@ def set_site
   end
 
   def build_pages_tree
-    tree = Page.where(site_id: @site.id).select(:id, :name, :parent_id, :position).hash_tree
+    tree = Page.where(site_id: @site.id).select(:id, :name, :parent_id, :position, :enabled).hash_tree
```

Comment:
we probably also need type here (for example, maps cannot be deleted, so the delete button needs to be dependent on the type)